deploy-chart.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -e -u -o pipefail
+
+NEXUS_USERNAME="${NEXUS_USERNAME:?NEXUS_USERNAME not specified or empty}"
+NEXUS_PASSWORD="${NEXUS_PASSWORD:?NEXUS_PASSWORD not specified or empty}"
+CHARTS_DIR="${1:-charts}"
+
+CHART_DIR="${CHARTS_DIR}/spinnaker"
+TMP_DIR="$(mktemp -d)"
+
+trap 'rm -rf -- "${TMP_DIR}"' INT TERM HUP EXIT
+helm3 package --dependency-update "${CHART_DIR}" --destination "${TMP_DIR}"
+find "${TMP_DIR}" -name '*.tgz' -exec \
+    curl --fail-with-body \
```

Comment:
`--fail-with-body` fails on any status code >= 400.

Reply:
Hmm. Ubuntu Focal (20.04) does not ships an older `curl` that does not have this option :disappointed: The newer Ubuntu Jammy LTS (22.04) that came out a two months ago does. Any ideas for an alternative by any chance? :sweat_smile: If not, not a blocker as I don't have push permissions to Nexus anyway.

Reply:
I pushed a few changes for this. Can you try it please? :)